app/src/renderer/styles/variables.styl

```diff
@@ -43,15 +44,16 @@ sans = -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubunt
 mono = Consolas, Monaco, monospace
 
 // alerts
-success = hsl(120,100%,50%)
+success = hsl(120,90%,50%)
 info = txt
-warning = hsl(60,100%,50%)
-danger = hsl(0,100%,50%)
+warning = hsl(52,100%,50%)
+danger = hsl(0,100%,40%)
 
 // sizes
 aw = 64rem
 width-main = 100%
 width-side = 16rem
+max-width = 600px
```

Comment:
Are we using this anywhere? And if we are, can it be a multiple of 1rem? (16px)

Reply:
using it for forms. yep. i can switch it rem's